argilla/docs/how_to_guides/annotate.md

```diff
@@ -0,0 +1,191 @@
+
+=== "By suggestions"
+
+    You can filter the records by the suggestions to the questions. Select the `Suggestions` dropdown, write the name of the question, if needed, and select it from the list. Then, select one of `Suggestion values`, `Score` or `Agent` when available. To filter using the suggested values, the rating questions will allow you to set a range of values and the label, multi-label and span questions will allow you to select the values you want to filter on. The text and ranking questions are not available for filtering. To filter by the score, you can set a range of values. To filter by the agent, you can select the agent you want to filter on.
+
+### Sort
+
+You can sort your records according to one or several attributes. To do so, you can click on the `Sort` button in the top left of the preferences view. A dropdown will appear with the available attributes to sort by. The time of insertion and last update are available for all the records. The suggestion scores, response and suggestion values for rating questions and metadata properties are available only when they were provided. To add a new attribute to sort by, you can click on the `Add another field` button. To remove an attribute, you can click on the `x` next to the attribute.
+
+## Annotate in team
+
+!!! info "Main use cases"
+    Multiple users on the same dataset need to be specially aligned on the task.
+
+!!! note
+    The 2.1 version of argilla will come with task distribution, which will allow you to assign records to specific users easily.
+
+### Access the annotation guidelines
+
+Before starting the annotation, we highly recommend consulting the annotation guidelines when available. If they have been specified, you can find them by clicking the `Annotation guidelines` button on the top right of the annotation view.
+
+### Edit guidelines in the settings
+
+You can edit the guidelines if you are `owner` or `admin` of the dataset. For that, you can click on the `Dataset settings` button on the the right of the top black bar. You will just need to write them or update them and save the changes.
+
+!!! tip
+    If you want further guidance on good practices for guidelines during the project development, check our [blog post](https://argilla.io/blog/annotation-guidelines-practices/).
+
+## See the progress
+
+!!! info "Main use cases"
+    You want to see the progress of the annotation task.
+
+### General progress view
+
+On the main page, before accessing the dataset, you can see the general progress of the annotation task. This is indicated in the `Global progress` column that shows the number of remaining records to be annotated and a progress bar. The progress bar displays the percentage and number of records submitted, conflicting (i.e. those with overlap), discarded and pending by hovering your mouse over it.
```

Comment:
Maybe this is a bit picky, but is there a reason why it used "left" and not "pending" aligned with the rest of the record status?

Reply:
Picky is fundamental! The global progress in dataset list is not exactly reflecting the records status we have into the dataset, Its more about providing a quick look on how its going the task. In any case with the task distribution V1 (Release 2.1) This Global progress will be probably simplified to Completed vs Left.